From 7fca3bb3635e3a460050a4ca90fca42bbc52292a Mon Sep 17 00:00:00 2001
From: Alejandro Mery <amery@geeks.cl>
Date: Tue, 29 May 2012 00:14:43 +0200
Subject: [PATCH 483/944] mmc: sunxi-host: don't use MMC_PM_IGNORE_PM_NOTIFY
 when not defined [EVIL!]

MMC_PM_IGNORE_PM_NOTIFY comes from android, and mmc_pm_io_shd_suspend_host()
has no side effects, but the logic may need to be reviewed.
---
 drivers/mmc/sunxi-host/host_op.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/mmc/sunxi-host/host_op.c b/drivers/mmc/sunxi-host/host_op.c
index 3d8c7c5..e9832e0 100644
--- a/drivers/mmc/sunxi-host/host_op.c
+++ b/drivers/mmc/sunxi-host/host_op.c
@@ -652,8 +652,10 @@ static int __devinit sunximmc_probe(struct platform_device *pdev)
     mmc->caps	    = MMC_CAP_4_BIT_DATA|MMC_CAP_MMC_HIGHSPEED|MMC_CAP_SD_HIGHSPEED|MMC_CAP_SDIO_IRQ;
     mmc->f_min 	    = 400000;
     mmc->f_max      = SMC_MAX_IO_CLOCK(pdev->id);
+#ifdef MMC_PM_IGNORE_PM_NOTIFY
     if (pdev->id==3 && !mmc_pm_io_shd_suspend_host())
         mmc->pm_flags = MMC_PM_IGNORE_PM_NOTIFY;
+#endif
 
     mmc->max_blk_count	= 4095;
     mmc->max_blk_size	= 4095;
-- 
1.8.0

